### ISSUE TITLE
bug: Unable to Share music over Instagram Story

### ISSUE THREAD

**Issue by Radical4ever** [ReVanced/revanced-patches#3346]
### Bug description

The issue is pretty straightfoward, its basically #480 and it still persist on this new versions
Steps:
-press share button when playing a music
-select instagram
-nothing happens

### Acknowledgements

- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.

**Comment by oSumAtrIX** [comment]
I cannot reproduce it, given the steps. Check on the unpatched app.

**Comment by oSumAtrIX** [comment]
Follow this and report back:

![image](https://github.com/ReVanced/revanced-patches/assets/13122796/e9430272-f9b7-4444-a851-3b7c6ff8e725)

**Comment by Radical4ever** [comment]
I was about to make that, im doin it 🤣

**Comment by Radical4ever** [comment]
Ok so i basically disabled all of them with only microG enabled and the issue still persist

**Comment by Radical4ever** [comment]
So, i guess is a microG problem?

**Comment by oSumAtrIX** [comment]
I highly doubt it. Leaving open for someone else to reproduce

**Comment by skypeias** [comment]
Yeah it never works for me. I click on the share button, press on Instagram icon, and nothing happens

**Comment by oSumAtrIX** [comment]
What are your environments? Device/ROM/Versions etc

**Comment by Radical4ever** [comment]
> What are your environments? Device/ROM/Versions etc

ok, so i use android 13 with OneUI 5 default rom that it come with the phone
